powerauth-java-server/src/main/java/io/getlime/security/powerauth/app/server/service/behavior/tasks/v3/CallbackUrlBehavior.java

```diff
@@ -306,6 +309,29 @@ public void notifyCallbackListenersOnOperationChange(OperationEntity operation)
         }
     }
 
+    /**
+     * Tries to asynchronously notify all operation callbacks that are registered
+     * for given application.
+     * 
+     * @param operations Operations to be notified about.
+     */
+    public void notifyCallbackListenersOnOperationChange(List<OperationEntity> operations) {
+        final List<OperationEntity> operationsWithAppId = operations.stream().filter(op -> op.getApplication() != null).collect(Collectors.toList());
+        final Set<Long> appIds = operationsWithAppId.stream().map(o -> o.getApplication().getId()).collect(Collectors.toSet());
+        final List<CallbackUrlEntity> callbackUrlEntities = callbackUrlRepository.findByApplicationIdInAndTypeOrderByName(appIds, CallbackUrlType.OPERATION_STATUS_CHANGE);
+        operationsWithAppId.stream().forEach(o -> {
+            callbackUrlEntities.stream().filter(cb -> cb.getApplicationId().equals(o.getApplication().getId())).forEach(cb -> {
+                Map<String, Object> callbackData = prepareCallbackDataOperation(cb, o);
+                try {
+                    notifyCallbackUrl(cb, callbackData);
+                } catch (RestClientException e) {
+                    // Log the error in case Rest client callback failed
+                    logger.error(e.getMessage(), e);
```

Comment:
Let's use `warn` level to avoid flooding the server with errors in case the remote backend is not running.